services/management/checks.go

```diff
@@ -96,7 +99,8 @@ func (s *ChecksAPIService) ListSecurityChecks() (*managementpb.ListSecurityCheck
 	res := make([]*managementpb.SecurityCheck, 0, len(checks))
 	for _, c := range checks {
 		_, disabled := m[c.Name]
-		res = append(res, &managementpb.SecurityCheck{Name: c.Name, Disabled: disabled})
+		desc := s.getCheckDescription(c)
+		res = append(res, &managementpb.SecurityCheck{Name: c.Name, Disabled: disabled, Description: desc})
```

Comment:
I'm not sure, but maybe it makes sense to extract description form checke when we dowload it, not each time when `ListSecurityChecks` invoked. I supose that `getCheckDescription` is expensive function.

Reply:
I think we should make description a top-level YAML field for exactly this reason: https://jira.percona.com/browse/SAAS-429
But that can be done later.